ignite/cmd/network_validator_set.go

```diff
@@ -0,0 +1,61 @@
+package ignitecmd
+
+import (
+	"errors"
+
+	"github.com/spf13/cobra"
+	profiletypes "github.com/tendermint/spn/x/profile/types"
+
+	"github.com/ignite/cli/ignite/pkg/cliui"
+)
+
+// NewNetworkValidatorSet creates a command to set an information in a validator profile
+func NewNetworkValidatorSet() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "set details|identity|website [value]",
+		Short: "Set an information in a validator profile",
+		Long: `Validators on Ignite can set a profile containing a description for the validator.
+The validator set command allows to set information for the validator.
+The following information can be set:
+- details: general information about the validator.
+- identity: piece of information to verify identity of the validator with a system like Keybase of Veramo.
+- website: website of the validator.
+`,
+		RunE: networkValidatorSetHandler,
+		Args: cobra.ExactArgs(2),
+	}
+	c.Flags().AddFlagSet(flagNetworkFrom())
+	c.Flags().AddFlagSet(flagSetHome())
+	c.Flags().AddFlagSet(flagSetKeyringBackend())
+	c.Flags().AddFlagSet(flagSetKeyringDir())
+	return c
+}
+
+func networkValidatorSetHandler(cmd *cobra.Command, args []string) error {
+	session := cliui.New()
+	defer session.Cleanup()
+
+	nb, err := newNetworkBuilder(cmd, CollectEvents(session.EventBus()))
+	if err != nil {
+		return err
+	}
+
+	n, err := nb.Network()
+	if err != nil {
+		return err
+	}
+
+	var validator profiletypes.Validator
+	switch args[0] {
+	case "details":
+		validator.Description.Details = args[1]
+	case "identity":
+		validator.Description.Identity = args[1]
+	case "website":
+		validator.Description.Website = args[1]
+	default:
+		return errors.New("invalid attribute, must provide details, identity or website")
+	}
```

Comment:
@lubtd as a validator, I want to pay less fees and set all my description details within a single tx :)

```sh
$ ignite n validator set
Need to provide at least --details, --identity or --website flags

$ ignite n validator set --details=foo --identity=bar --website=baz
```

Reply:
Yes, I find it nicer to have an option to set a single value without using flag
Maybe we can have both option under the same `set` command

Reply:
yes I agree it's a little ugly with flags, I don't know if there a known pattern for this usecase...

Reply:
Feel free to change or to keep it